### PR TITLE
NovaBreath Keep card 

### PR DESCRIPTION
**Pull request by cvennel**
Copied the Keep card abstract class I think from @kaubster PR #73.

Since this card only modifies the targets of attack rolls no trigger logic is necessary, just check if attacker owns this card

### CODE REVIEW

**Comment by cvennel** [comment]
#82

**Comment by cvennel** [comment]
#123

**Comment by cvennel** [comment]
closes #123

File: backend/game/cards/keep_card.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+    def immediate_effect(self, player_that_bought_the_card, other_players):
+        player_that_bought_the_card.add_card(self)
+
+    @abstractmethod
+    def special_effect(self, player_that_bought_the_card, other_players):
+        raise NotImplementedError
+
+    def discard_effect(self, player_that_bought_the_card, other_players):
+        player_that_bought_the_card.remove_card(self)
```
Comment (gpfaia) [inline backend/game/cards/keep_card.py]: NIT: Best practice is to have a new line at EOF

**Pull request merged**